src/storage/txn/process.rs

```diff
@@ -181,50 +175,46 @@ impl<E: Engine> Executor<E> {
         if let Some(term) = cb_ctx.term {
             task.cmd.mut_context().set_term(term);
         }
-        let pool = self.take_pool();
+        let sched_pool = self.take_pool();
+        let engine = sched_pool.engine;
         let readonly = task.cmd.readonly();
-        pool.schedule(move |ctx: &mut SchedContext<E>| {
+        sched_pool.pool.spawn(move || {
             fail_point!("scheduler_async_snapshot_finish");
 
-            let _processing_read_timer = ctx
-                .processing_read_duration
-                .with_label_values(&[tag])
-                .start_coarse_timer();
+            let read_duration = tikv_util::time::Instant::now_coarse();
```

Comment:
can we use `Instant::now_coarse()` here?

Reply:
fixed it.